src/datapump/sync/sync.py

```diff
@@ -168,23 +168,21 @@ class IntegratedAlertsSync(Sync):
         "umd_glad_sentinel2_alerts",
         "wur_radd_alerts",
     ]
-    MULTI_BAND_CALC = "np.ma.array([(A > 20000) * (A != 30000) * (A < 40000) * (2 * (A - (20000 + (A>30000) * 10000)) + (A<30000) * 1), (B > 20000) * (B != 30000) * (B < 40000) * (2 * (B - (20000 + (B>30000) * 10000)) + (B<30000) * 1), (C > 20000) * (C != 30000) * (C < 40000) * (2 * (C - (20000 + (C>30000) * 10000)) + (C<30000) * 1)])"
-    SINGLE_BAND_CALC = "np.ma.array(np.ma.sum([np.clip(np.ma.sum([(np.ma.sum([(np.ma.masked_equal(A,0)>0)*1,(np.ma.masked_equal(B,0)>0)*1,(np.ma.masked_equal(C,0)>0)*1],axis=0)>=2)*4,(np.ma.sum([np.ma.masked_equal(A,0)&1,np.ma.masked_equal(B,0)&1,np.ma.masked_equal(C,0)&1],axis=0)==0)*3,(np.ma.sum([np.ma.masked_equal(A,0)&1,np.ma.masked_equal(B,0)&1,np.ma.masked_equal(C,0)&1],axis=0)==1)*2],axis=0),None,4).filled(0)*10000,np.ma.min([np.ma.masked_equal(A,0)>>1,np.ma.masked_equal(B,0)>>1,np.ma.masked_equal(C,0)>>1],axis=0)],axis=0).filled(0))"
+    INPUT_CALC = "np.ma.array(((((A.data % 10000)  > 0) | ((B.data % 10000) > 0) | ((C.data % 10000)   > 0)) * (10000 + 10000 * np.where(((A.data // 10000) + (B.data // 10000) + (C.data // 10000)) > 3, 3, np.maximum(((A.data // 10000) + (B.data // 10000) + (C.data // 10000)) - 1, 0)).astype(np.uint16) + (65535 - np.maximum.reduce([(((A.data%10000)>0)*((65535-(A.data%10000)).astype(np.uint16))),(((B.data%10000)>0)*((65535-(B.data%10000)).astype(np.uint16))),(((C.data%10000)>0)*((65535-(C.data%10000)).astype(np.uint16)))])))), mask=False)"
```

Comment:
@dmannarino as a prettier string:
`"""
        np.ma.array(
        (
            (((A.data % 10000) > 0) | ((B.data % 10000) > 0) | ((C.data % 10000) > 0))
            * (
                10000
                + 10000
                * np.where(
                    ((A.data // 10000) + (B.data // 10000) + (C.data // 10000)) > 3,
                    3,
                    np.maximum(
                        ((A.data // 10000) + (B.data // 10000) + (C.data // 10000)) - 1,
                        0,
                    ),
                ).astype(np.uint16)
                + (
                    65535
                    - np.maximum.reduce(
                        [
                            (
                                ((A.data % 10000) > 0)
                                * ((65535 - (A.data % 10000)).astype(np.uint16))
                            ),
                            (
                                ((B.data % 10000) > 0)
                                * ((65535 - (B.data % 10000)).astype(np.uint16))
                            ),
                            (
                                ((C.data % 10000) > 0)
                                * ((65535 - (C.data % 10000)).astype(np.uint16))
                            ),
                        ]
                    )
                )
            )
        ),
        mask=False,
    )
    """`

Reply:
nvm, github removes the formatting in comments anyway...